docs/release-notes.md

```diff
@@ -1,45 +1,41 @@
+--8<-- "References/abbreviations.md"
 # Release Notes
 
 If you haven't had a chance - checkout the [Letter From Devs](./letter.md) for a message from our development team.
-## Key Feature Changes for 1.2
-This section outlines key features changes between versions 1.1 and 1.2 for [Open Active Management Technology (Open AMT) Cloud Toolkit.](Glossary.md#o) 
+## Key Feature Changes for 1.3
+This section outlines key features changes between versions 1.2 and 1.3 for Open AMT Cloud Toolkit. 
 
 ### Additions
-#### [RPS](Glossary.md#r)
-- **Hostname and Device Metadata tags:** We have added the ability to define metadata tags for a each Intel&reg; AMT profile.  These tags will be applied to each device that is configured using the profile.  Additionally, if RPC sends up the Intel&reg; AMT device OS hostname during activation, RPS will save this information as part of the device information.  When RPS configures a device, it will perform a POST call to MPS to register that device in MPS.  This works in conjunction with the improved metadata gathering that RPC added in 1.1.
-- **Node 14 LTS support:** RPS has been updated to support running on Node 14 LTS. 
+#### RPS & MPS
+- **Stateless Authentication:** The RESTful APIs for both RPS and MPS now utilizie stateless authentication tokens to validate user access.  With this change, our reference deployment now utilizes Kong API Proxy that validates the token and then fowards authenticated requests to either MPS or RPS.  This simplifies how API calls are made into the Open AMT Cloud Toolkit services by putting the service name that is being called into the URL.  
 
-#### [RPC](Glossary.md#r)
-- **Docker support added:** Added a dockerfile to RPC that allows customers to build and execute RPC in a docker container on the Intel&reg; AMT device.
+    ```html
+    https://example.url/mps
+    https://example.url/rps
+    ```
+    
+    With the addition of Kong and its rich plug-in support, this opens up options for integrators to utilize a broader set of user authentication services.  For example, the same tokens that are already being used by the rest of a device management tool to authenticate users can also be used to authenticate users with the Open AMT Cloud Toolkit service(s).  This new feature should make integration of Open AMT Cloud Toolkit microservices into existing management consoles much more seamless.
```

Comment:
I'd link to the list of auth plugins for folks to see what can be leveraged. And additionally call out that MPS `/authorization` endpoint now provides a JWT token that can be used.

Reply:
Updated with links to Kong's Plug-in Hub and added reference to MPS /authorization endpoint